### DISCUSSION TITLE
sbt document out fo sync?

### DISCUSSION THREAD

**Discussion by ghost** [sbt/sbt#6945] [Q&A]
![image](https://user-images.githubusercontent.com/278153/176809546-6158d701-7405-4965-8c42-0b677714c8af.png)

per the dropdown select list, looks it only 0.13

what if the current sbt document ? latest means what?

**Comment by SethTisue** [comment]
latest means 1.x, currently 1.6.2. choosing that item on the dropdown takes you to https://www.scala-sbt.org/1.x/docs/ which are the latest docs

I agree it would be clearer if the dropdown said "Latest version (1.x)" instead of just "Latest version". I assume a PR making that change would be accepted.
